src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml

```diff
@@ -35,12 +35,20 @@
             <converters:MarginConverter x:Key="MarginConverter" />
         </Grid.Resources>
         <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
             <RowDefinition Height="5" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
+        <TextBlock TextWrapping="Wrap">
+            <Run Text="{x:Static local:GridOptionPreviewControl.CodeStylePageHeader}"/>
+            <Run Text=" "/>
```

Comment:
@jasonmalinowski could you take a look at this? Thanks

Reply:
Doing a space this way is what I'd prefer actually since I think it's a better semantic representation of "what we want". Imagine you're a screen reader processing this. The space it'll know is a space and is a word break.

Reply:
@jasonmalinowski Thanks, I was just asking since it differed from another instance of a similar feature in the project. 👍

Reply:
@sharwell I wouldn't assume in this case that one had a reason for it.